tutorials/ring_background.ipynb

```diff
@@ -0,0 +1,415 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ring Background Estimation\n",
+    "\n",
+    "## Context:\n",
+    "Classically, image analysis in IACT have often relied on the ring background model to get a background estimate. This method typically involves extracting a ring around a trial source position to estimate the background strength. To correct for the varying camera acceptance within the field of view, an acceptance correction function is used to compute the normalisation for each position of the ring. To read more about this method, see [here.](https://arxiv.org/abs/astro-ph/0610959)\n",
+    "\n",
+    "## Proposed approach:\n",
+    "\n",
+    "The analysis workflow is roughly\n",
+    " - Compute the sky maps keeping each observation separately using the `Analysis` class\n",
```

Comment:
Since you have to perform a loop over images anyway, maybe using the HLI is not needed here.

Reply:
I thought so initially as well, but without the HLI, it means exposing the `DataStore`, the `SafeMaskMaker` etc, which seems to distract the main focus of the notebook. So, it seemed clearer to stick with the HLI, and add an extra loop. 

Of course, I can change this if you prefer.

Reply:
OK. I guess we will have to update to a new solution as soon as the HLI is adapted to the ring background.